tests/unit/console/bumper_ci_service_test.ts

```diff
@@ -0,0 +1,167 @@
+import { Rhum } from "../../../mod.ts";
```

Comment:
mod.ts?

Reply:
yea i don't see why rhum can reference itself. should it not?

Reply:
im an idiot, i think i thought i was looking at the services repo, so i was confused why a dependency was being imported from mod.ts - you were right to question my stupidity!